src/ResourceManager/HDInsight/Commands.HDInsight.Test/ScenarioTests/DataLakeStoreScenarioTests.cs

```diff
@@ -10,11 +10,18 @@ public DataLakeStoreScenarioTests(Xunit.Abstractions.ITestOutputHelper output)
             ServiceManagemenet.Common.Models.XunitTracingInterceptor.AddToContext(new ServiceManagemenet.Common.Models.XunitTracingInterceptor(output));
         }
 
-        [Fact]
+        [Fact (Skip = "Test requires fixing GraphAPI by upgrading to latest test framework")]
```

Comment:
@srsiva could clarify why this test is being skipped. Is it test framework issue or GraphApi issue.
Also make sure you put comment on the test itself as to why is this being skipped and what is the plan to enable it back again.
But at this time, it is not clear why is this being skipped.